apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx

```diff
@@ -70,6 +71,30 @@ class TeacherDashboardHeader extends React.Component {
       />
     ));
 
+  progressNotSavingNotification() {
+    let details = (
+      <p>
+        {i18n.disabledProgress1()} {i18n.disabledProgressTeacherDashboard2()}{' '}
+        <a
+          target="_blank"
+          rel="noopener noreferrer"
+          href={disabledBubblesSupportArticle}
+        >
+          {i18n.learnMore()}
+        </a>
+      </p>
+    );
+
+    return (
+      <Notification
+        type={NotificationType.failure}
+        notice={i18n.disabledProgressTeacherDashboard1()}
+        details={details}
```

Comment:
I don't think I have a strong opinion either way, but is there a reason to do the "Learn more" as an inline link to the article rather than a button? Mostly asking since the `Notification` component comes with the option to use a button built in, which to my memory is more commonly used.

Reply:
Thanks for the tip - I switched it to use the `detailsLink` props because the button looked weird with such a long notification text.